api/v1/model_handlers.go

```diff
@@ -272,6 +274,20 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Param(ws.QueryParameter("end", "End time for requested metric").DataType("string")).
 		Writes(MetricResult{}))
 
+	// The /sum/namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name} endpoint exposes
+	// an aggregated metric for a list od pods of the model.
+	ws.Route(ws.GET("/sum/namespaces/{namespace-name}/pods/{pod-list}/metrics/{metric-name}").
```

Comment:
`/namespaces/{namespace-name}/pods/sum/metrics/{metric-name}` will also match to 
`/namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name}` causing path ambiguity.

Reusing /namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name} will require a change in the result type in this (an other paths for consistency). Are you OK with that?